Update behavioral.mdx (#16)
Proyecto-Nutria/otter-guides: i18n/es/docusaurus-plugin-content-docs/current/interview/interviewee/behavioral.mdx

title: Preguntas de la forma de ser
---

> En esta parte de la entrevista, el entrevistador querrá saber cómo te comportas bajo determinadas situaciones basándose en tu experiencia pasada mediante preguntas del tipo:

Reviewer: ```suggestion
En esta parte de la entrevista el entrevistador querrá saber cómo te comportas en determinadas 
situaciones basándote en tu experiencia pasada mediante preguntas del tipo:
```

- "Cuéntame sobre una vez cuando..."
- "Dame un ejemplo de..."
- "¿Alguna vez has..."
- "Describe una instancia en la que tuviste que..."

#### Algunos ejemplos completos:

- Cuéntame sobre una vez cuando te enfrentaste a un problema que tenía varias soluciones posibles. ¿Cuál era el problema y cómo determinaste el curso de acción? ¿Cuál fue el resultado de esa elección?

- ¿Cuándo tomaste un riesgo, cometiste un error o fallaste? ¿Cómo respondiste y cómo creciste a partir de esa experiencia?

- Describe un momento en el que lideraste un proyecto.

- ¿Qué hiciste cuando necesitabas motivar a un grupo de personas o fomentar la colaboración en un proyecto específico?

- ¿Cómo has aprovechado los datos para desarrollar una estrategia?

> ### Consejos para responder preguntas conductuales de manera efectiva

Reviewer: ```suggestion
### Consejos para responder preguntas sobre tu forma de ser de manera efectiva
```

- Usa el método [STAR](https://www.indeed.com/career-advice/interviewing/how-to-use-the-star-interview-response-technique) para responder a las preguntas.

> - Echa un vistazo a esta lista más extensa de preguntas de ejemplo: [Las 30 preguntas más comunes en entrevistas conductuales para ingenieros de software | Manual de entrevistas técnicas](https://www.techinterviewhandbook.org/behavioral-interview-questions/).

Reviewer: ```suggestion
- Echa un vistazo a esta lista más extensa de ejemplos: [The 30 most common Software Engineer behavioral interview questions | Tech Interview Handbook](https://www.techinterviewhandbook.org/behavioral-interview-questions/).
```

- Prepara tus respuestas para las preguntas más populares. No las memorices, pero recuerda la situación que deseas presentar para esa pregunta específica.

> - Sé consciente del tiempo que utilizas para responder cada pregunta; trata de evitar la redundancia.

Reviewer: ```suggestion
Sé consciente del tiempo que utilizas para responder cada pregunta, evita ser redundante.
```
